Electronic_UserDB.sql

```diff
@@ -22,6 +22,7 @@ as
 begin
 update UserDB set username=@username, password=@password,active=@active where userId=@userId
 end
```

Comment:
Why are you updating username?  We only need to update the password here. Please focus on naming convention